beacon-chain/sync/validate_beacon_blocks.go

```diff
@@ -93,6 +94,9 @@ func (s *Service) validateBeaconBlockPubSub(ctx context.Context, pid peer.ID, ms
 
 	// Verify the block is the first block received for the proposer for the slot.
 	if s.hasSeenBlockIndexSlot(blk.Block().Slot(), blk.Block().ProposerIndex()) {
+		if features.Get().EnableEquivocatingBlockLogging {
+			log.WithField("block", hexutil.Encode(msg.Data)).Debug("equivocating block detected")
```

Comment:
This doesn't church that it's an equivocation, unless I'm missing something this would dox the node